src/Catalyst.Abstractions/Kvm/Models/BlockParameters.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using Nethermind.Core.Json;
+
+namespace Catalyst.Abstractions.Kvm.Models
+{
+    public class BlockParameter
+    {
+        public static BlockParameter Earliest = new BlockParameter(BlockParameterType.Earliest);
+
+        public static BlockParameter Pending = new BlockParameter(BlockParameterType.Pending);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'Pending' or make it 'const' or 'readonly'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4411902)